src/main/java/geekbrains/java2/ht1b/Main.java

```diff
@@ -0,0 +1,257 @@
+     4) Учитываем что при установке крестика или нолика место должно быть пустым, в противном случае необходим запрос на повторный ход;
+     5) Делаем проверку на победу одного из игроков;
+     6) Сделать несколько вариантов поведения АИ:
+     Компьютерный игрок случайно ставит свои метки, либо продумывает свои ходы (блокирует линию игрока, и продолжает свою)
+     [b]Дополнительные условия(сложность увеличена)[/b]:
+     1) Программа реализуется в виде отдельных классов;
+     2) Размер поля и необходимой серии можно указывать(например поле 5х5 и серия 4 фишки в ряд)
+     3) Когда поле больше чем 3х3, компьютерный игрок должен стать намного умнее, иначе у него не будет шансов;
+
+     */
+
+    //глобальные переменные
+    static String[][] board = {{"A1", "A2", "A3"}, {"B1", "B2", "B3"}, {"C1", "C2", "C3"}};
+    static Scanner scanner = new Scanner(System.in);
+
+
+    public static void main( String[] args ){
+        System.out.println("Игра в крестики-нолики");
+        printBoard();
+
+        while(computerMove() != "Больше ходов нет. Ничья"){
+            System.out.println("Ваш ход");
+            updateBoard(playerMove(), "1");
+            printBoard();
+            if(winGame() == true){
+                System.out.println("Вы выиграли!");
+                break;
+            }
+
+            System.out.println("Ход компьютера");
+            System.out.println("Компьютер ходит: " + computerMove());
+            updateBoard(computerMove(), "2");
+            printBoard();
+            if(winGame() == true){
+                System.out.println("Вы проиграли!");
+                break;
+            }
+
+        }
+        System.out.println("Игра закончена");
+
+    }
+
+    //получаем значения в ячейках на доске
+    private static String getValue(String value){
+        if(value == "1"){
+            return " X";
+        } else if(value == "2"){
+            return " O";
+        } else {
+            return value;
+        }
+    }
+
+    //Выводим поле в консоль
+    private static void printBoard(){
+
+        System.out.println("----------------");
+        for (int i = 0; i < board.length; i++){
+            System.out.print("| ");
+            for(int j = 0; j < board[i].length; j++){
+                System.out.print(getValue(board[i][j]) + " | ");
+
+            }
+            System.out.println();
+            System.out.println("----------------");
+        }
+
+
+    }
+
+
+    //Ход игрока
+    private static String playerMove(){
+        String playerMove;
+
+        do{
+            playerMove = scanner.nextLine();
+            if (validMove(playerMove) == false)
+            {
+                System.out.println("Некорректное значение, выберите другое.");
+            }
+        } while(validMove(playerMove) == false);
+        return playerMove;
+    }
+
+    //Некорректный ход
+    private static boolean validMove(String move){
+        if(move.equalsIgnoreCase("A1") && (board[0][0] == "A1"))
```

Comment:
\[Checkstyle\] ERROR: String literal expressions should be on the left side of an equalsIgnoreCase comparison\.